.github/workflows/bb-forge-test.yml

```
@@ -0,0 +1,161 @@
+      FOUNDRY_INVARIANT_DEPTH: ${{ inputs.foundry-invariant-depth }}
+      FOUNDRY_INVARIANT_RUNS: ${{ inputs.foundry-invariant-runs }}
+      FOUNDRY_FUZZ_RUNS: ${{ inputs.foundry-fuzz-runs }}
+      FOUNDRY_PROFILE: ${{ inputs.foundry-profile }}
+      MAINNET_RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Check out the repo"
+        uses: "actions/checkout@v4"
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.10)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)